### PR TITLE
Commented out broken test

### PR DESCRIPTION
**Pull request by nirname**
## :bookmark_tabs: Summary

Broken test blocks our development pipeline

Related #3274

## :straight_ruler: Design Decisions

Commented the test, left link to the issue inside the code. Let me know if I we can simply remove the test.

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [x] :computer: have added necessary unit/e2e tests.
- [x] :notebook: have added documentation. Make sure [`MERMAID_RELEASE_VERSION`](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/community/development.md#3-update-documentation) is used for all new features.
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: cypress/integration/rendering/gantt.spec.js [hunk 1]
```diff
@@ -520,15 +520,23 @@ describe('Gantt diagram', () => {
...
+  it.skip('should render a gantt diagram with very large intervals, skipping excludes if interval > 5 years', () => {
     imgSnapshotTest(
       `gantt
         title A long Gantt Diagram
         dateFormat   YYYY-MM-DD
         axisFormat   %m-%d
         tickInterval 1day
         excludes     weekends
-
+
```
Comment (aloisklink) [inline cypress/integration/rendering/gantt.spec.js:539]:
```suggestion

```

I'm surprised the linter didn't spot this, but we might want to remove this whitespace change before merging.

**Review by aloisklink — APPROVED** [review]
Looks good to me!

Thanks for spotting this issue, we've really got to get better at not breaking CI. Maybe we should forbid pushing to `develop`/`main` and only allow using GitHub PRs.

**Comment by sidharthv96** [comment]
[release/10.5.0](https://github.com/mermaid-js/mermaid/tree/release/10.5.0) was already broken, few commits before 63f4a56ec556aa729b465f4d40c8078b2f65cdd0. 
I didn't realise the test was failing as it ran properly locally, so thought the failure was with the release docs, which should've been fixed before release. 
I actually saw this failure later when checking develop, fixed the release docs, and assumed the OOM was a transient issue. Didn't notice which test was failing.

**Comment by nirname** [comment]
No problem guys, that is ok. Thank you for quick response

**Pull request merged**